### DISCUSSION TITLE
Welcome to pycontrails Discussions!

### DISCUSSION THREAD

**Discussion by mlshapiro** [contrailcirrus/pycontrails#1] [Announcements]
## 👋 Welcome!

We will use Discussions as a place to connect with members of the community. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Discuss model improvements.
  * Engage with other community members.

**Comment by bpicard-fluctus** [comment]
Hi !
I'm Bruno Picard. I'm working on atmospheric humidity observations, focusing on measurements by microwave radiometers.  I'm an independent expert providing support to space agencies.

Thank you for your work, it's very useful, also connected to a study I'm working on.

I have selected profiles from the 
[NWP-SAF dataset](https://nwp-saf.eumetsat.int/site/software/atmospheric-profile-data/)

and I have applied the the pycontrails ISSR and SAC methods on it.
I'm selecting profiles for altitudes between 8 km and 12 km: I raise a flag if at least 5 levels have issr or sac  flags == 1

I've found that sometime, the sac flag can be raised when ISSR is not.
Is it an expected behaviour ?
I was expecting that sac flag would only be raised on ISRR regions ?

but I may have missing a point, this is not exactly my topic of expertise.

thanks again for your work,
best regards,
Bruno

**Reply by zebengberg** [reply]
Hi Bruno!

Interestingly, the SAC can be satisfied even in completely dry air. See, for example, the [right hand column on page 10](https://elib.dlr.de/32128/1/mz-96.pdf) (the paragraphing starting with "Fig. 4a shows T_LC ...").

We have a third [PCR model](https://py.contrails.org/api/pycontrails.models.pcr.html) which simply computes the intersection of ISSR and SAC (ie, both are satisfied). This could be what you're seeking.

**Reply by bpicard-fluctus** [reply]
Hi Zeb,

that's totally clear now.
Thanks for the reference and PCR is exactly what I will need !
